lidar_camera_calibration_py/src/config_utils.py

```
@@ -0,0 +1,55 @@
+
+def parse_config_file(config_path: str) -> dict[str, Any]:
+    """Parse the main configuration file for calibration parameters.
+
+    Args:
+        config_path (str): Path to the configuration file.
+
+    Returns:
+        dict[str, Any]: Dictionary of configuration parameters.
+    """
+    params: Dict[str, Any] = {}
+    with open(config_path, 'r') as f:
```

Ruff (0.14.0)

Unnecessary mode argument

Remove mode argument

(UP015)